charts/node-server/templates/mongodb-deploy.yaml

```diff
@@ -0,0 +1,59 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mongodb
+  namespace: "tensorleap"
+  labels:
+    app: mongodb
+spec:
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+  selector:
+    matchLabels:
+      app: mongodb
+  template:
+    metadata:
+      labels:
+        app: mongodb
+    spec:
+      containers:
+        - name: mongodb
+          image: mongo:5.0.11
+          args:
+            - --dbpath
+            - /data/db
+          imagePullPolicy: "IfNotPresent"
+          ports:
+            - name: mongodb
+              containerPort: 27017
+          livenessProbe:
+            failureThreshold: 6
+            initialDelaySeconds: 30
+            periodSeconds: 20
+            successThreshold: 1
+            timeoutSeconds: 10
+            exec:
+              command:
+                - mongosh --eval 'db.adminCommand("ping")'
+          readinessProbe:
+            failureThreshold: 6
+            initialDelaySeconds: 5
+            periodSeconds: 10
+            successThreshold: 1
+            timeoutSeconds: 5
+            exec:
+              command:
+                - mongosh --eval 'db.hello().isWritablePrimary || db.hello().secondary' | grep -q 'true'
+          resources:
+            limits: {}
+            requests: {}
+          volumeMounts:
+            - name: datadir
+              mountPath: /data/db
```

Comment:
As I tried to explain, IIRC the `datadir` is currently the directory that contains `data/db`
So you can use a different directory (with the `--dbpath` argument to `mongod`) or use the `subPath` as well

Reply:
Fixed